CI/PHP-CS-Fixer/example/ILIAS_CodeStyle.xml

```diff
@@ -0,0 +1,333 @@
+<code_scheme name="ILIAS Code Style" version="173">
```

Comment:
Maybe we should rename the file or create a sub-directory to identify this is a file for `PHPStorm`

Reply:
@legionth & @mjansenDatabay any concrete suggestions for folder or file name?

Reply:
`CI/PHP-CS-Fixer/example/PhpStorm/ILIAS_CodeStyle.xml`

`CI/PHP-CS-Fixer/example/PhpStorm_ILIAS_CodeStyle.xml`

?

Reply:
I'm preferring the first.

Reply:
Me too, done.

Reply:
Option 1 please.